packages/lesswrong/lib/collections/jargonTerms/views.ts

```diff
@@ -0,0 +1,24 @@
+import JargonTerms from "./collection"
+import { ensureIndex } from '../../collectionIndexUtils';
+
+declare global {
+  interface JargonTermsViewTerms extends ViewTermsBase {
+    view?: JargonTermsViewName
+    postId?: string
+    rejected?: boolean
+    forLaTeX?: boolean
+  }
+}
+
+JargonTerms.addDefaultView((terms: JargonTermsViewTerms) => {
+  return {
+    selector: { deleted: false },
+  };
+});
+
+JargonTerms.addView("jargonTerms", function (terms: JargonTermsViewTerms) {
+  return {
+    selector: { postId: terms.postId },
+    options: { sort: { forLaTeX: -1, term: 1, createdAt: 1 } }
```

Comment:
You probably don't care about sorting by `forLaTeX: -1` given that's not even being used yet, right?